Mediator/FarmerMediator.java

```diff
@@ -0,0 +1,65 @@
+package com.clearfaun.mdlag;
+
+import java.util.ArrayList;
+
+/**
+ * Created by SpencerDepas on 6/29/16.
+ */
+public class FarmerMediator implements Mediator{
+
+    private ArrayList<Colleague> colleagues;
+
+
+
+    public FarmerMediator(){
+
+        colleagues = new ArrayList<Colleague>();
+
+    }
+
+
+    @Override
+    public void sellHighestPricedAnimal(int count) {
```

Comment:
Remember single responsibility. Your method should do only one thing: one responsibility. This method is doing many things. The solution is refactoring.